x/bank/expected_keepers.go

```diff
@@ -1,10 +0,0 @@
-package bank
-
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
-// expected crisis keeper
-type CrisisKeeper interface {
```

Comment:
Did we get rid of the whole `expected_keepers.go` concept?

Reply:
no, we just nolonger require this one expected keeper within bank, these where used to register invariance routes, but all that logic has been abstracted from the modules to the module manager

Reply:
👍